lib/configuration.dart

```diff
@@ -34,6 +36,24 @@ class Configuration {
   bool useAppSetIdForDeviceId;
   /// Web specific
   String? appVersion;
+  /// Web specific
+  String? deviceId;
```

Comment:
device Id can be set in iOS and Android too right? Similary app version too. 

Can we group config by platform?

Cross platform first, then specific platforms next?